MSAL/src/native_auth/controllers/factories/MSALNativeAuthResultFactory.swift

```diff
@@ -71,33 +69,14 @@ final class MSALNativeAuthResultFactory: MSALNativeAuthResultBuildable {
                                 accountClaims: jsonDictionary)
     }
 
-    func makeAuthTokens(tokenResult: MSIDTokenResult, context: MSIDRequestContext) -> MSALNativeAuthTokens? {
-        guard let refreshToken = tokenResult.refreshToken as? MSIDRefreshToken else {
-            MSALLogger.log(
-                level: .error,
-                context: context,
-                format: "Refresh token invalid, account result could not be created")
-            return nil
-        }
-        return MSALNativeAuthTokens(accessToken: tokenResult.accessToken,
-                                              refreshToken: refreshToken,
-                                              rawIdToken: tokenResult.rawIdToken)
-    }
-
     func makeUserAccountResult(tokenResult: MSIDTokenResult, context: MSIDRequestContext) -> MSALNativeAuthUserAccountResult? {
         let account =  makeAccount(tokenResult: tokenResult, context: context)
-        guard let authTokens = makeAuthTokens(tokenResult: tokenResult, context: context) else {
-            MSALLogger.log(
-                level: .error,
-                context: context,
-                format: "Auth Tokens could not be created")
-            return nil
-        }
-        return .init(account: account, authTokens: authTokens, configuration: config, cacheAccessor: cacheAccessor)
+        let rawIdToken = tokenResult.rawIdToken
```

Comment:
Do we need this here? We can just have tokenResult.rawIdToken in the next line

Reply:
Updated ;)